### PR TITLE
Update how-to-aovpn-conditional-access.md

### PR DESCRIPTION
**Pull request by SanDeo-MSFT**

### CODE REVIEW

**Review by anaharris-ms — APPROVED** [review]

File: WindowsServerDocs/remote/remote-access/how-to-aovpn-conditional-access.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ When a user attempts a VPN connection, the VPN client makes a call into the Web
 The VPN client then sends the certificate issued by Azure AD to the VPN for credential validation. 
 
 > [!NOTE]
-> Azure AD uses the most recently created certificate in the VPN connectivity blade as the Issuer.
+> Azure AD uses the most recently created certificate in the VPN connectivity blade as the Issuer. Azure AD Conditional Access VPN connection leaf certificates now support strong certificate mappings, a Certificate-based authentication requirement introduced by [KB5014754](https://support.microsoft.com/topic/kb5014754-certificate-based-authentication-changes-on-windows-domain-controllers-ad2c23b0-15d8-4340-a468-4d4f3b188f16). VPN connection leaf certificates now include a [SID extension](https://learn.microsoft.com/openspecs/windows_protocols/ms-wcce/e563cff8-1af6-4e6f-a655-7571ca482e71) of (1.3.6.1.4.1.311.25.2), which contains an encoded version of the user’s SID obtained from the onPremisesSecurityIdentifier attribute.
```
Comment (anaharris-ms) [inline WindowsServerDocs/remote/remote-access/how-to-aovpn-conditional-access.md:76]:
```suggestion
> Azure AD uses the most recently created certificate in the VPN connectivity blade as the Issuer. Azure AD Conditional Access VPN connection leaf certificates now support strong certificate mappings, a certificate-based authentication requirement introduced by [KB5014754](https://support.microsoft.com/topic/kb5014754-certificate-based-authentication-changes-on-windows-domain-controllers-ad2c23b0-15d8-4340-a468-4d4f3b188f16). VPN connection leaf certificates now include a [SID extension](https://learn.microsoft.com/openspecs/windows_protocols/ms-wcce/e563cff8-1af6-4e6f-a655-7571ca482e71) of (1.3.6.1.4.1.311.25.2), which contains an encoded version of the user’s SID obtained from the onPremisesSecurityIdentifier attribute.
```

**Comment by SanDeo-MSFT** [comment]
#sign-off

**Comment by SanDeo-MSFT** [comment]
#sign-off

**Comment by Jak-MS** [comment]
#assign: @robinharwood, @Xelu86 

@robinharwood @Xelu86 
Can you review the proposed changes? 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by anaharris-ms** [comment]
#sign-off

**Pull request merged**